src/main/java/com/rocketden/main/controller/v1/UserSocketController.java

```diff
@@ -1,30 +1,34 @@
 package com.rocketden.main.controller.v1;
 
-import com.rocketden.main.dto.user.UserDto;
 import java.util.HashSet;
 import java.util.Set;
 
+import com.rocketden.main.dto.user.UserDto;
+
+import org.springframework.messaging.handler.annotation.DestinationVariable;
 import org.springframework.messaging.handler.annotation.MessageMapping;
 import org.springframework.messaging.handler.annotation.SendTo;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class UserSocketController extends BaseRestController {
 
-    Set<UserDto> users = new HashSet<>();
+    private static final String SEND_TO_PATH = BaseRestController.BASE_SOCKET_URL + "/{roomId}/subscribe-user";
```

Comment:
Where is the `roomId` variable? I don't see where it is actually set in this file?

Reply:
It comes from the MessageMapping string and is mapped to a variable using `@DestinationVariable String roomId`, and then when the SendTo annotation occurs, it should replace `{roomId}` with the correct string.